packages/holocron/src/moduleRegistry.js

```diff
@@ -30,8 +31,12 @@ function getModuleBlockList() {
   return moduleBlockList;
 }
 
+function addHigherOrderComponent(module) {
```

Comment:
Need to expose this decoration function to allow for `updateModuleRegistry` to operate for the serverside wrapping of Modules with `holocronModule`.